docs_outdoornav_user_manual/integration_requirements/platform_computer_checklist.mdx

```diff
@@ -6,18 +6,19 @@ toc_min_heading_level: 2
-| <input type="checkbox"/> | On the platform computer, run `ping cpr-outdoor` and confirm it is successful.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
-| <input type="checkbox"/> | On the platform computer, [Install ROS Noetic](http://wiki.ros.org/noetic/Installation/Ubuntu) (the `ros-noetic-ros-base` version is sufficient).                                                                                                                                                                                                                                                                                                                                                                         |
-| <input type="checkbox"/> | On the platform computer, configure ROS Noetic so it is running with the `ROS_MASTER_URI` set to the base platform computer. Note that `localhost` is acceptable only if the primary subnet on the platform computer is the `192.168.131.XXX` subnet.                                                                                                                                                                                                                                                                     |
-| <input type="checkbox"/> | On the platform computer, complete all checks in [Interface Control Checklist](/docs_outdoornav_user_manual/integration_requirements/interface_control_requirements) and confirm that all listed ROS topics are publishing as expected.                                                                                                                                                                                                                                                                                   |
+| <input type="checkbox"/> | On the platform computer, add the following line to the `/etc/hosts` file: `192.168.131.5 cpr-<project_code>-onav>`                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Should we add a "<br/> after "file: "?

Reply:
not sure what you mean by this comment

Reply:
Interesting. It took my html syntax and rendered it directly rather than showing the text. Should we force the file content onto a new line by adding:
`<br/>`